client/packages/system/src/Vaccination/Components/VaccinationModal.tsx

```diff
@@ -91,7 +96,9 @@ export const VaccinationModal = ({
           }
         }
       } catch (e) {
+        setIsSaving(false);
         console.error(e);
+        error(t('error.something-wrong'))();
```

Comment:
Second side quest - there was no loading or error indication in the vaccination modal! was a quick add so threw it in 🙈